index.md

```diff
@@ -261,6 +261,67 @@ mediator.on('user-create', () => {});
 
 IE9+ and all modern browsers
 
+## Using Themes
+We deliver some predefined themes (e.g. shepherd-theme-default or shepherd-theme-square). You are welcome to use them and embed them into your page as additional stylesheets.
+
+```html
+<head>
+  <link rel="stylesheet" href="shepherd-theme-default.css">
+</head>
+```
+
+Individual customizations to the standard themes must be made within the CSS file. Furthermore, individual CSS classes can be passed via the tour instance or for each step individually, to overwritte or supplement  existing style properties.
+
+```javascript
+let tour = new Shepherd.Tour({
+  defaults: {
+    classes: 'shepherd-theme-custom'
+  }
+});
+```
+
+### Custom Themeing
+We use [SASS](https://sass-lang.com/) as pre-processor for CSS. In connection with SASS there are extensive possibilities to generate CSS. For example, SASS can calculate or increase the saturation of color values. In addition, variables can be defined (similar to a scripting language), which ultimately end up as values in the CSS result. We make use of these extended possibilities by extracting themeing-relevant values as variables (__variables.scss_). This makes it easy to individualise colors and shapes.
```

Comment:
`(__variables.scss_)` should be `(__variables.scss__)` so that all following content isn't bold